data/meshes/inline-quad.mesh

```diff
@@ -0,0 +1,7 @@
+MFEM INLINE mesh v1.0
+
+type = quad
+nx = 4
+ny = 4
+sx = 1.0
+sy = 1.0
```

Comment:
I don't think this interferes with the MFEM mesh reader but can we get a newline here?